models2/CommentModel.es6.js

```diff
@@ -2,6 +2,8 @@ import RedditModel from './RedditModel';
 import Record from '../apiBase/Record';
 import { COMMENT, COMMENT_LOAD_MORE } from './thingTypes';
 
+import votable from './mixins/votable';
```

Comment:
I think this is fine. I'm hesitant to add decorators to node-build yet as the spec is in flux ([more info](http://babeljs.io/docs/plugins/transform-decorators/)). From what I've read the new spec isn't completely backwards-compatible, and adding the babel-plugin for the 'legacy' syntax can interact with the static properties plugin in obscure, error'ing, ways. It might be better now so maybe thats worth re-considering